decidim-elections/app/views/decidim/elections/votes/_election_votes_question.html.erb

```diff
@@ -26,6 +26,20 @@
 
     <%= render("election_votes_modal", answer: answer) if more_information?(answer) %>
   <% end %>
+
+  <% if question.min_selections == 0 %>
+    <%= label_tag nil, nil, class: "evote__check evote__card" do %>
+      <% if question.max_selections == 1 %>
+        <%= radio_button_tag "question_#{question.id}", "nota_input_#{question.id}", false, class: "nota_input" %>
```

Comment:
mm, did you consider creating the NOTA input as a regular answer input with `""` value (or similar)? Maybe that would help us to keep the JS almost as simple as now, using the existing support for `min_selection`.

Reply:
I think @slickepinne has made a great proposal by handling minimal number of responses instead of handling a blank response. I think I found it clever in terms of code. 
On the other hand, the idea of ​​having a blank answer is also valid.